src/main/kotlin/no/nav/familie/ef/mottak/service/SøknadskvitteringService.kt

```diff
@@ -54,6 +55,15 @@ class SøknadskvitteringService(
         return motta(søknadDb, vedlegg)
     }
 
+    @Transactional
+    fun mottaArbeidsøkerSkjema(skjemaForArbeidssøker: SkjemaForArbeidssøker): Kvittering {
```

Comment:
Kan noe ala dette fungere slik de over?
```suggestion
    fun mottaArbeidsøker(søknad: SøknadMedVedlegg<SøknadArbeidsøker>): Kvittering {
```

Reply:
Det er en egen type i den gamle flyten og kontrakter, eventuelt så kan det jo være en egen oppgave

Reply:
Skulle hatt dobbel-s i navnet her `mottaArbeidssøkerSkjema`. Tenker også at det går fint å inkludere i navnet at der et arbeidssøkerskjema som blir mottatt - det er jo et skjema vi mottar